### ISSUE TITLE
ROTATE_DOOR not working on v1.4.0

### ISSUE THREAD

**Issue by krystiancharubin** [mrlt8/docker-wyze-bridge#362]
```
NET_MODE: LAN
ROTATE_DOOR: "true"
RTSP_READTIMEOUT: 30
QUALITY: HD180
ENABLE_AUDIO: "true"
```
```
ffmpeg -loglevel verbose -fflags +genpts+flush_packets+nobuffer -flags low_delay -analyzeduration 50 -probesize 50 -f h264 -i pipe: -f s16le -ar 8000 -i /tmp/front-door.wav -c:v copy -c:a aac -filter:a volume=5 -movflags +empty_moov+default_base_moof+frag_keyframe -f tee -map 0:v -map 1:a -shortest [select=v,a:f=rtsp:rtsp_transport=tcp]rtsp://0.0.0.0:8554/front-door
```

**Comment by mrlt8** [comment]
Thank you! I see it!

**Comment by mrlt8** [comment]
Should be fixed in v1.4.1!
